Cargo.toml

```diff
@@ -85,6 +85,7 @@ smol_str = "0.3"
 tracing = { version = "0.1.40", default-features = false }
 
 [dev-dependencies]
+font8x8 = "0.3.1"
```

Comment:
Nit: Seems like this hasn't been updated in 4 years. Maybe there's something a bit more maintained we can use?

Reply:
I considered it, but because it's "just" a font, I figured it probably wasn't that important that it's unmaintained 🤔 Maybe the decision between different debugging/embedded bitmap font crates could better be made based off of license or looks?

Actually, I think I found a potential snafu: the font data font8x8 is based on says it's public domain, but the maintainer/author seems to located in Germany, and there isn't a workaround license like CC0 to make up for the fact that "speech act" public domain dedications don't work under German law? This is almost certainly not a problem, but it COULD be one, hypothetically. Maybe I'll open an issue on it. <https://github.com/dhepper/font8x8>